scheduler/server/orig_scheduling_alg_test.go

```diff
@@ -23,9 +308,9 @@ func Test_StatefulSchedulerRequestorCounts(t *testing.T) {
 		{"numTasks": "1", "requestor": "p2Requestor", "priority": "2"},
 	}
 	jobIds := addRequestorTestJobsToScheduler(phase1Profiles, s)
-	s.step()                          // start the first set of tasks
-	time.Sleep(50 * time.Millisecond) // let task threads run to complete
-	s.step()                          // get first set of completed tasks and start next set
+	s.step()                           // start the first set of tasks
+	time.Sleep(500 * time.Millisecond) // let task threads run to complete
```

Comment:
This seems like a pretty large increase. Do you know why that is needed?

Reply:
removed file